CHANGELOG.md

```diff
@@ -5,11 +5,11 @@
 ### Bugfixes
 - [#2479](https://github.com/influxdata/kapacitor/pull/2479): Fix: influx gzip writes for large writes.
 - [#2488](https://github.com/influxdata/kapacitor/pull/2488): Fix function node name for ServiceNow handler so it is properly camelcased. 
-- [#2489](https://github.com/influxdata/kapacitor/pull/2489): Fix memory leaks in JoinNode and UnionNode.
+- [#2489](https://github.com/influxdata/kapacitor/pull/2489): Fix: memory leaks in JoinNode and UnionNode.
```

Comment:
curious why add 2 colons?

Reply:
was a typo, should have gotten rid of the one with an extra instead of adding one.